readme.md

```diff
@@ -12,10 +12,12 @@ pipeline:
     input: openapi-document/multi-api/identity
   modelerfour/new-transform:
     input: modelerfour
-  csharp-v3:
+  type-identifier:
     input: modelerfour/new-transform
-  csharp-v3/emitter:
-    input: csharp-v3
+  model-creator:
```

Comment:
I think it might still be good to have these be csharp specific because in isolation I have no idea this is generating csharp code.

Reply:
Once I have actual plugins clearly defined, that is the plan. For now, they are just temporary names.